src/CouchDB.Driver/CouchDB.Driver.csproj

```diff
@@ -32,9 +32,8 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <PackageReference Include="Flurl.Http" Version="3.0.1" />
-    <PackageReference Include="Humanizer.Core" Version="2.8.26" />
-    <PackageReference Include="Microsoft.CodeAnalysis.FxCopAnalyzers" PrivateAssets="All" Version="3.3.1" />
```

Comment:
Any reason why the analyzer is gone?

Reply:
I had updated all my nuget packages (hence Flurl and Humanizer changing as well), and the FxCop analyzers newer than 3.3.1 have been deprecated, so I removed that at the same time. Happy to leave it back in though if you prefer.

See https://docs.microsoft.com/en-us/visualstudio/code-quality/migrate-from-legacy-analysis-to-net-analyzers?view=vs-2022